common/djangoapps/student/scope_resolver.py

```diff
@@ -57,6 +57,65 @@ def resolve(self, scope_name, scope_context, instance_context):
         )
 
 
+class NamespaceEnrollmentsScopeResolver(NotificationUserScopeResolver):
+    """
+    Implementation of the NotificationUserScopeResolver abstract
+    interface defined in edx-notifications.
+
+    We will be passed in a namespace (aka course_id) in the context
+    and we must return a Django ORM resultset or None if
+    we cannot match.
+    """
+
+    def resolve(self, scope_name, scope_context, instance_context):
+        """
+        The entry point to resolve a scope_name with a given scope_context
```

Comment:
Can you add some info about 'instance_context', which is not currently mentioned in the docstring?